Assets/MixedRealityToolkit/Inspectors/Profiles/BaseMixedRealityProfileInspector.cs

```diff
@@ -101,7 +124,7 @@ private static bool RenderProfileInternal(SerializedProperty property, GUIConten
 
                 if (!renderedProfile.IsCustomProfile && profile.IsCustomProfile)
                 {
-                    if (GUILayout.Button(new GUIContent("</>", "Replace with a copy of the default profile."), EditorStyles.miniButton, GUILayout.Width(32f)))
+                    if (GUILayout.Button(new GUIContent("Clone", "Replace with a copy of the default profile."), EditorStyles.miniButton, GUILayout.Width(42f)))
```

Comment:
Did you pick 42 cause it was the answer to life, the universe, and everything? 😉

Reply:
Of course.

Reply:
@Ecnassianer brought up a [good comment](https://github.com/Microsoft/MixedRealityToolkit-Unity/issues/3605#issuecomment-472139263). 

I'm thinking "clone" or something along those lines makes more sense.

Reply:
I went with that suggestion.